apps/website/src/pages/docs/[...slug].astro

```diff
@@ -0,0 +1,21 @@
+---
+import { getCollection } from 'astro:content';
+import Layout from './_layout.astro';
+
+export async function getStaticPaths() {
+  return (await getCollection('docs')).map((entry) => {
+    let slug;
+    if (entry.slug) {
```

Comment:
Just curious, how are the types for `entry.slug` generated?

![image](https://github.com/iTwin/iTwinUI/assets/45748283/7a044381-ccec-4528-b67a-45f70de5bdb9)

Reply:
it's done automatically by astro, but it's not important (at least i don't find the types useful). there is some info here: https://docs.astro.build/en/guides/content-collections/#the-astro-directory